pkgs/applications/misc/gnuradio/osmosdr.nix

```
@@ -1,23 +1,30 @@
-{ stdenv, fetchgit, cmake, pkgconfig, boost, gnuradio, rtl-sdr, uhd
-, makeWrapper, hackrf, airspy
+{ stdenv, fetchgit, cmake, pkgconfig, makeWrapper
+, boost
 , pythonSupport ? true, python, swig
+, airspy
+, gnuradio
+, hackrf
+, rtl-sdr
+, soapysdr-with-plugins
+, uhd
 }:
 
 assert pythonSupport -> python != null && swig != null;
 
 stdenv.mkDerivation rec {
   name = "gnuradio-osmosdr-${version}";
-  version = "0.1.4";
+  version = "2018_08_15";
```

Can we change that to dashes?
```suggestion
  version = "2018-08-15";
```